### ISSUE TITLE
fix colab download for netcdf files in ICMW

### ISSUE THREAD

**Issue by slayoo** [atmos-cloud-sim-uj/PySDM#254]

**Comment by slayoo** [comment]
This seems to be related to #253 as the error (shown after clicking any download button) is:
```
TraitError: The 'description' trait of a Button instance must be a unicode string, but a value of None <class 'NoneType'> was specified.
```

Indeed, button descriptions and icons are blank on Colab

**Comment by slayoo** [comment]
addressed in #265
